crates/bevy_gizmos/src/primitives/dim3.rs

```diff
@@ -944,6 +944,55 @@ impl<T: GizmoConfigGroup> Drop for Torus3dBuilder<'_, '_, '_, T> {
     }
 }
 
+// ramp
+
+impl<'w, 's, T: GizmoConfigGroup> GizmoPrimitive3d<Ramp> for Gizmos<'w, 's, T> {
+    type Output<'a> = () where Self: 'a;
+
+    fn primitive_3d(
+        &mut self,
+        primitive: Ramp,
+        position: Vec3,
+        rotation: Quat,
+        color: impl Into<Color>,
+    ) -> Self::Output<'_> {
+        if !self.enabled {
+            return;
+        }
+
+        let [half_extend_x, half_extend_y, half_extend_z] = primitive.half_size.to_array();
```

Comment:
Why are we splitting these up?

Reply:
Just for convenience when working with them. It's nicer to write `half_extend_x` than `primitive.half_size.x` everytime.

Reply:
But couldn't you do this?
```
        // transform the points from the reference unit cube-like ramp to the actual ramp coords
        let [a, b, c, d, e, f] = [
            [-1.0, -1.0, -1.0],
            [-1.0, -1.0, 1.0],
            [1.0, -1.0, -1.0],
            [1.0, -1.0, 1.0],
            [1.0, 1.0, 1.0],
            [-1.0, 1.0, 1.0],
        ]
        .map(|s| Vec3::from_array(s) * primitive.half_size)
        .map(rotate_then_translate_3d(rotation, position));
```
There's no need to split them from what I can see

Reply:
Fair enough :)